test/dogear.test.js

```diff
@@ -243,82 +240,82 @@ describe('Plugin reports on paths.', (tap) => {
 
 describe('Plugin reports ops.', (tap) => {
 
-  tap.test('Starts listening to ops reports.', (t) => {
+  tap.test('Starts listening to ops reports.', async t => {
     t.plan(1);
 
-    const server = initServer();
+    const server = await initServer();
 
-    server.register({
-      register: plugin,
+    await server.register({
+      plugin,
       options: {
         statsdConfig: {
           mock: true
         },
         opsInterval: 500
       }
-    }).then(() => {
+    });
 
-      const plugin = server.plugins['dogear'];
+    const dogear = server.plugins['dogear'];
 
-      t.ok(plugin._ops);
+    t.ok(dogear._ops);
 
-      server.stop({ timeout: 0 }).then(t.end);
-    });
+    await server.stop({ timeout: 0 });
+    t.end();
   });
 
 
-  tap.test('Report ops metrics.', (t) => {
-    t.plan(2);
+  tap.test('Report ops metrics.', async t => {
+    t.plan(3);
 
-    const server = initServer();
+    const server = await initServer();
 
-    server.register({
-      register: plugin,
+    await server.register({
+      plugin,
       options: {
         statsdConfig: {
           mock: true
         },
-        opsInterval: 500
+        opsInterval: 10
       }
-    })
-    .then(() => {
+    });
 
-      const client = server.statsd;
+    const client = server.statsd;
+    const gaugeSpy = sandbox.spy(client, 'gauge');
 
-      sandbox.stub(client, 'gauge', (metric, value) => {
+    // wait for the first data sample by oppsy
+    await new Promise(resolve => setTimeout(resolve, 20));
 
-        t.type(metric, 'string');
-        t.type(value, 'number');
+    t.ok(gaugeSpy.called);
+    t.type(gaugeSpy.getCall(0).args[0], 'string');
+    t.type(gaugeSpy.getCall(0).args[1], 'number');
 
-        server.stop({ timeout: 0 }).then(t.end);
-      });
-    });
+    await server.stop({ timeout: 0 });
+    t.end();
   });
 
 
-  tap.test('Ignores unsupported ops metric.', (t) => {
+  tap.test('Ignores unsupported ops metric.', async t => {
     t.plan(1);
 
-    const server = initServer();
+    const server = await initServer();
 
-    server.register({
-      register: plugin,
+    await server.register({
+      plugin,
       options: {
         statsdConfig: {
           mock: true
         },
         opsInterval: 500,
-        opsMetrics: [ 'bad.op.metric' ]
```

Comment:
Unnecessary styling change.

Reply:
Please prevent this by eslint rules, so you won't have to tell everybody who's creating a PR for you. Changed it anyway.